tests/unit/amo/components/TestRatingManager.js

```diff
@@ -398,10 +397,11 @@ describe(__filename, () => {
 
     const root = render({ store });
 
-    const message = root.find(RatingManagerNotice);
-    expect(message).toHaveProp('message', 'Star rating saved');
-    expect(message).toHaveProp('type', successType);
-    expect(message).toHaveProp('hideMessage', false);
+    const notice = root.find(RatingManagerMessage);
+    expect(notice).toHaveProp('hideMessage', false);
+    expect(shallow(notice.prop('message'))).toHaveText(
+      '<Icon /> Star rating saved',
```

Comment:
This seems fragile to me since it relies on Enzyme's debug output (showing the nested components in the shallow wrapper). I guess it's probably fine but really the test seems like it's checking the text. You could do that directly:

```diff
diff --git a/tests/unit/amo/components/TestRatingManager.js b/tests/unit/amo/components/TestRatingManager.js
index 6fcd1f4ba..4f56c5f9b 100644
--- a/tests/unit/amo/components/TestRatingManager.js
+++ b/tests/unit/amo/components/TestRatingManager.js
@@ -399,8 +399,8 @@ describe(__filename, () => {
 
     const notice = root.find(RatingManagerMessage);
     expect(notice).toHaveProp('hideMessage', false);
-    expect(shallow(notice.prop('message'))).toHaveText(
-      '<Icon /> Star rating saved',
+    expect(shallow(notice.prop('message')).text()).toContain(
+      'Star rating saved',
     );
   });
 
```

You could then add an `expect(wrapper.find(Icon)).toHaveLength(1)` but I'm not sure that it's necessary.